action/deployment_add.go

```diff
@@ -88,7 +94,9 @@ EXAMPLES:
     $ {{.HelpName}} --org MyOrg --repo MyRepo --ref 48afb5bdc41ad69bf22588491333f7cf71135163
   5. Add a deployment for a repository with a specific description.
     $ {{.HelpName}} --org MyOrg --repo MyRepo --description 'my custom message'
-  6. Add a deployment for a repository when config or environment variables are set.
+  6. Add a deployment for a repository with two parameters.
+    $ {{.HelpName}} --org MyOrg --repo MyRep --parameter 'key=value' --parameter 'foo=bar'
```

Comment:
Think this is a typo

```suggestion
    $ {{.HelpName}} --org MyOrg --repo MyRepo --parameter 'key=value' --parameter 'foo=bar'
```

Reply:
Updated in the latest commit.